gmsh_interop/runner.py

```diff
@@ -190,6 +190,7 @@ def __init__(
             gmsh_executable: str = "gmsh",
             output_file_name: str | None = None,
             target_unit: Literal["M", "MM"] | None = None,
+            save_output_file_in: str | None = None,
```

Comment:
Could you document what these are all for? `output_file_name` at least sounds redundant with `save_output_file_in`? Could you also document `save_tmp_files_in`?

Reply:
Added docs. As previously implemented, `output_file_name` is the base filename only (no path), i.e., it sets the name of the output mesh file inside the temporary working dir. For compatibility's sake I left it alone and added a separate argument to save the mesh file to a different path.